### ISSUE TITLE
Select non-standard layouts such as those from yFiles

### ISSUE THREAD

**Issue by kieran-mace** [tmuetze/Bioconductor_RCy3_the_new_RCytoscape#45]
Even though I've installed the yFiles app, and can apply their layouts from Cytoscape, I cannot access the layout options from `getLayoutNames(cw)`

**Comment by AlexanderPico** [comment]
That's correct. This is not a bug. The Cytoscape license for yFiles lets us expose them only via the UI. The license doesn't allow them to be exposed programmatically.

Longer answer from yFiles developer (copied from Cytoscape Helpdesk): 

> " It's true that for the app on the cytoscape app store there is no technical and legal option to expose the layouts provided by our library through cytoscape to third parties.
> That doesn't mean, though that you cannot use the yFiles layouts at all.
> The layouts come as part of a generic programming library and as such can be used from any program if you have a license for the (commercial) yFiles library.
> That would mean that you could create your own version of the yFiles layouts app for your specific Cytoscape application. The fact that this would give you full access to the huge and powerful programming API of yFiles would also mean that you can do a lot more than what the current App allows you to do: You would be able to configure the layout algorithms to a very high-degree: You could apply different settings for each and every element in your diagram, configure whether you are interested in speed or quality, etc.
> You would not be allowed to create an application that provides these services to third parties, though; so you could not create another yFiles app, just without that limitation. Our standard license terms do not allow wrapping of our APIs, so you would only be allowed to use it for your application."
